src/lib.rs

```diff
@@ -357,7 +358,7 @@ pub struct Package {
     pub id: PackageId,
     /// The source of the package, e.g.
     /// crates.io or `None` for local projects.
-    pub source: Option<Source>,
+    pub source: Option<Source>, // TODO(aatifsyed): should this be RegistryName?
```

Comment:
here

Reply:
Yea, I think we can remove the `Source` type now

Reply:
Having seen `Source::is_crates_io`, it's not the same as RegistryName